### ISSUE TITLE
iPad crash at launch (EXC_CORPSE_NOTIFY) for Zebra newer than 1.1.22

### ISSUE THREAD

**Issue by Kyogre** [zbrateam/Zebra#1962]
**Describe the bug**
On ipad Zebra crashes immediately after opening. Affects only versions newer than 1.1.22.

**Device Information:**
 - iOS Version: 14.8
 - Zebra Version: 1.1.23, 1.1.24, 1.125
 - Device Type: iPad 2017
 - Jailbreak Type: odysseyra1n

**Crash Log or System Log**
https://pastebin.com/Mcu6fqiA

**Comment by absidue** [comment]
Thanks for reporting the crash, could you please install Cr4shed with a different package manager and provide us the crashlog from inside Cr4shed. It will hopefully have even more useful information to help debug the issue.

**Comment by Kyogre** [comment]
thank you! Cr4shed is very useful. It found the culprit of the crash immediately. It was Tweakio. And you know what helped me to fix the crash? I just opened Tweakio settings, changed Tab bar to Search bar, opened Zebra (no crash), opened Tweakio settings again and changed style back to Tab bar, opened Zebra and still no crash and Tweakio works just fine.

I updated the crash log, the link is the same, but I don't think there is something to fix after we found a workaround.
